tests/authstore.js

```diff
@@ -0,0 +1,130 @@
+import test from 'brittle'
```

Comment:
so far just writing out tests rather than using a strategy runner that takes json as input. the api was changing too much for that. i think it would be best to create an issue for adding a strategy runner later.